src/test/java/com/openlattice/rehearsal/datasets/DataSetsTest.kt

```diff
@@ -0,0 +1,107 @@
+package com.openlattice.rehearsal.datasets
+
+import com.openlattice.organization.OrganizationIntegrationAccount
+import com.openlattice.organizations.Organization
+import com.openlattice.rehearsal.authentication.MultipleAuthenticatedUsersBase
+import junit.framework.Assert
+import org.junit.BeforeClass
+import org.junit.Test
+import java.sql.DriverManager
+import java.sql.SQLException
+import java.util.*
+
+
+open class DataSetsTest : MultipleAuthenticatedUsersBase() {
+    companion object {
+        lateinit var credentials: OrganizationIntegrationAccount
+        lateinit var organization: Organization
+        lateinit var organizationID: UUID
+        lateinit var url: String
+
+        const val create_query = "CREATE TABLE IF NOT EXISTS " +
```

Comment:
Does this work? I think some special configuration might need to happen in order to allow multiple queries in one statement, but I could definitely be wrong about that.